id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/RestRequestBuilder.java

```diff
@@ -78,9 +101,9 @@ public RestRequestDTO buildRequest(RestServicesConstants restService, Object req
 
 		String serviceName = restService.getServiceName();
 
-		String uri = env.getProperty(serviceName.concat(REST_URI));
-		String httpMethod = env.getProperty(serviceName.concat(REST_HTTP_METHOD));
-		String timeout = env.getProperty(serviceName.concat(REST_TIMEOUT));
+		String uri = (mapBuilder.containsKey(serviceName) && mapBuilder.get(serviceName).containsKey(REST_URI))?mapBuilder.get(serviceName).get(REST_URI):null;

```

Comment:
These statements seems to have same logic. please create a method for this and invoke that in all places by passing the parameters accordingly.